### DISCUSSION TITLE
Metadata issue.

### DISCUSSION THREAD

**Discussion by ChiefMedicalOfficer** [advplyr/audiobookshelf#319] [Q&A]
Is anyone else having issues with the metadata agents?

All 3 are returning 'No Results' for me.

I wanted to check here first before I confirm if it's my setup or not.

**Comment by ChiefMedicalOfficer** [comment]
DNS issue. Container needed a down/up.

**Reply by Kadaj81** [reply]
@ChiefMedicalOfficer have the same issue.  How did you solve it?
